app/helpers/repositories_helper.rb

```diff
@@ -34,13 +34,11 @@ def access_change_hint
   end
 
   def access_options
-    t('repository.access.options').select do |k,v|
-      if @repository.mirror?
-        k.to_s.split('_')[1] == 'r'
-      else
-        true
-      end
-    end.invert
+    t('repository.access.options').invert
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.